src/index.html

```diff
@@ -4,14 +4,104 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no,
+        initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      autocomplete="off">
+      <fieldset class="blocks">
+        <legend>Personal information</legend>
+        <lable class="forms">
+          Surname:
+          <input type="text" name="surname"
+            minlength="4" maxlength="30">
+        </lable>
+        <lable class="forms">
+          Name:
+          <input type="text" name="name">
+        </lable>
+        <lable class="forms">
+          How old are You?
+          <input type="number"
+            name="old"
+            min="1" max="100"
+            value="12">
+        </lable>
+        <lable class="forms">
+          Full date of birth:
+          <input type="date" name="birth">
+        </lable>
+        <lable class="forms">
+          I accept the term of the agreement
+          <input type="checkbox" name="term"
+            id="agree">
+        </lable>
+      </fieldset>
+      <fieldset class="blocks">
+        <legend>Registration</legend>
+        <lable class="forms">
```

Comment:
add one empty line between same-level elements
```suggestion
        <legend>Registration</legend>
        
        <lable class="forms">
```